### PR TITLE
Test/usage text

### PR DESCRIPTION
**Pull request by murrayholmes96**
Sorry about bad branch name pushed this as a test but seemingly works

Scottish-Natural-Heritage/Licensing#478

### CODE REVIEW

File: cypress/integration/003-usage.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ describe('usage page directly', function () {
     cy.get('h1').should('contain', 'there is a problem with the service');
   });
 });
-
```
Comment (MikeCoats) [inline cypress/integration/003-usage.js:7]: Not sure why this line needed to be removed...

File: cypress/integration/003-usage.js [hunk 2]
```diff
@@ -25,20 +24,18 @@ describe('usage page ', function () {
...
     cy.url().should('include', '/trap-registration-number');
   });
-
   it('no button should navigate to trap no-usage page', function () {
     cy.visit('/usage');
     cy.get('#main-content form input[type="radio"][value="no"]').click();
     cy.get('#main-content form button.naturescot-forward-button').click();
     cy.url().should('include', '/no-usage');
   });
-
```
Comment (MikeCoats) [inline cypress/integration/003-usage.js:35]: ...or these ones. I'm pretty sure they were there to visually distinguish separate tests. Did this get flagged by the linter?

File: src/views/usage.njk [hunk 3]
```diff
@@ -1,15 +1,15 @@
-{% extends "_base.njk" %}
-
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+-{% extends "_base.njk" %}
+-
+-{% from "govuk/components/button/macro.njk" import govukButton %}
+-{% from "govuk/components/radios/macro.njk" import govukRadios %}
+-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
```
Comment (MikeCoats) [inline src/views/usage.njk:5]: Not sure about those `-` at the start of every line. Pretty sure they're an accidental addition.

**Review by MikeCoats — CHANGES_REQUESTED** [review]
A couple of wee questions and something I think is a copy/paste error.

**Comment by murrayholmes96** [comment]
The errors are from copy and pasting, hopefully fixed now

File: cypress/integration/003-usage.js (hunk 1 above)
Comment (MikeCoats) [reply cypress/integration/003-usage.js]: Resolved in 911481548a8e2d58a41903741679c291d257b844.

File: cypress/integration/003-usage.js (hunk 2 above)
Comment (MikeCoats) [reply cypress/integration/003-usage.js]: Resolved in 911481548a8e2d58a41903741679c291d257b844.

File: src/views/usage.njk (hunk 3 above)
Comment (MikeCoats) [reply src/views/usage.njk]: Resolved in 911481548a8e2d58a41903741679c291d257b844.

**Review by MikeCoats — APPROVED** [review]

**Pull request merged**